### PR TITLE
Cad 2 crud

### PR DESCRIPTION
**Pull request by ViktorasSce**
CRUD items

### CODE REVIEW

File: src/Application/Services/ItemService.cs [hunk 1]
```diff
@@ -2,82 +2,104 @@
...
-            Address = userEntity.Address,
-        };
+            throw new NotFoundException("No items found");
+        }
+
+        IEnumerable<ItemDto> items = itemEntities.Select(itemEntity => new ItemDto
+        {
+            Key = itemEntity.Key,
+            Value = JsonSerializer.Deserialize<List<string>>(itemEntity.Value),
+            ExpirationPeriod = itemEntity.ExpirationPeriod
```
Comment (arturascervinskas) [inline src/Application/Services/ItemService.cs]: add expiration date

File: src/Application/Services/ItemService.cs [hunk 2]
```diff
@@ -2,82 +2,104 @@
...
-        users = usersEntities.Select(o => new Item()
+        ItemDto item = new()
         {
-            Id = o.Id,
-            Name = o.Name,
-            Address = o.Address,
-        }).ToList();
+            Key = itemEntity.Key,
+            Value = JsonSerializer.Deserialize<List<string>>(itemEntity.Value),
+            ExpirationPeriod = itemEntity.ExpirationPeriod
```
Comment (arturascervinskas) [inline src/Application/Services/ItemService.cs]: add expiration date

File: src/DBEditor/DBEditor.csproj
Comment (arturascervinskas) [inline src/DBEditor/DBEditor.csproj:1]: revert changes

File: src/DBEditor/Properties/launchSettings.json
Comment (arturascervinskas) [inline src/DBEditor/Properties/launchSettings.json:1]: revert changes

**Review by arturascervinskas — APPROVED** [review]
Good job mldc

File: src/Application/Dto/ItemDto.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Application.Dto;
+
+public class ItemDto
+{
+    public string Key { get; set; }
+    public List<string> Value { get; set; }
+    public int ExpirationPeriod { get; set; }
```
Comment (ghost) [inline src/Application/Dto/ItemDto.cs]: should be nullable

File: src/Application/Dto/GetItemDto.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Application.Dto;
+
+public class GetItemDto
```
Comment (ghost) [inline src/Application/Dto/GetItemDto.cs]: Should be Item

File: src/Application/Dto/ItemDto.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Application.Dto;
+
+public class ItemDto
```
Comment (ghost) [inline src/Application/Dto/ItemDto.cs]: Should be ItemCreate

**Review by arturascervinskas — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**